scripts/do-release.sh

```diff
@@ -67,15 +67,18 @@ function _update_go_mod() {
 
 function update_go_mod() {
     local target="$1"
-    if [[ ! -f projects/${project}/go.mod ]]; then
-        return 1
-    fi
 
-    # Run in subshell so we don't change the working directory even on failure
-    (
-        cd "projects/${project}"
-        dryrun _update_go_mod
-    )
+    shopt -s globstar
+    for gomod in projects/"${project}"/**/go.mod; do
```

Comment:
This has the possibility to find go.mod files in vendor directory, perhaps `find` with exclusion filter is better (or other exclusion method)?

Reply:
`go mod vendor` no longer downloads `go.mod` files for dependencies: https://go.dev/doc/go1.17#go-command

Reply:
Nice, good to know :)